pexec/managed_process_unix.go

```diff
@@ -126,6 +127,14 @@ func (p *managedProcess) kill() (bool, error) {
 	return forceKilled, nil
 }
 
+// forceKill kills everything in the process group. This will not wait for completion and may result in a zombie process.
+func (p *managedProcess) forceKill() error {
+	pidStr := strconv.Itoa(-p.cmd.Process.Pid)
+	p.logger.Infof("killing entire process group %d", p.cmd.Process.Pid)
```

Comment:
```suggestion
	p.logger.Infof("killing entire process group of process %d", p.cmd.Process.Pid)
```
ID is process ID not process group ID.

Reply:
the process group id is the process id, because of https://github.com/viamrobotics/goutils/pull/399#discussion_r1901046413